conda_build/cli/main_skeleton.py

```diff
@@ -46,6 +49,13 @@ def execute(args):
     parser, args = parse_args(args)
     config = Config(**args.__dict__)
 
+    deprecated.topic(
+        "3.25.0",
+        "4.0.0",
+        topic="`conda skeleton` and `conda-skeleton`",
+        addendum="Use the `grayskull` project instead.",
```

Comment:
Should we update all of these addendums to say something more informative:

```
Use the standalone `grayskull` project, see https://github.com/conda/grayskull.
```

Reply:
This is effectively what we have in the help blurb so I'm ok with not stressing over this and leaving the warnings as-is.

Reply:
Yeah, let's keep it in the help